packages/esm-patient-search-app/src/config-schema.ts

```diff
@@ -22,7 +22,6 @@ export const configSchema = {
         'Disable the default "keyup search" for instant patient search as typing concludes on tablet devices',
     },
     searchFilterFields: {
-      _type: Type.Object,
```

Comment:
Isn't the `type` still valid here?

Reply:
Wow everything about this section of the config schema is totally nuts. I think I need to rewrite the documentation. Something is really not making sense to people. I can't believe this is passing the config system's validation. Maybe I should rewrite that too.

Reply:
@harshthakkr could you attempt to fix this section, now that (if you read my other comments) you understand `Type.Object` and the difference between elements and trees? You are correct to get rid of all these `Type.Object`s; none of them are correct. Make sure `_default` is defined at the right level etc.